Source/SpeechToTextV1/WebSockets/RecognitionSettings.swift

```diff
@@ -31,7 +31,7 @@ public struct RecognitionSettings: Codable, Equatable {
     /// The format of the audio data. Endianness is automatically detected by the Speech to Text
     /// service. For more information aboutthe supported formats, visit:
     /// https://console.bluemix.net/docs/services/speech-to-text/input.html#formats
```

Comment:
This link does not appear to resolve to the information on audio formats. You probably want to change this to

https://console.bluemix.net/docs/services/speech-to-text/audio-formats.html

Reply:
Good catch!